packages/api/start/routes/chat.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable jsdoc/check-tag-names */
+/* eslint-disable jsdoc/check-indentation */
+const Route = use('Route');
+
+Route.get('chat', 'ChatController.show').middleware(['auth']);
+
+Route.get('chat/:chatId/messages', 'ChatController.getMessages').middleware(['auth']);
+
+Route.post('chat/:chatId/messages', 'ChatController.postMessage')
```

Comment:
I'm suggesting this just to maintain the project convention
This change would also need to be reflected in the controller

```suggestion
Route.get('chat/:id/messages', 'ChatController.index').middleware(['auth']);

Route.post('chat/:id/messages', 'ChatController.store')
```